src/internal/components/chart-popover/index.tsx

```diff
@@ -128,19 +139,20 @@ function ChartPopover(
         keepPosition={true}
         allowVerticalOverflow={true}
         allowScrollToFit={isPinned}
+        hoverArea={true}
```

Comment:
the hover-area style was moved to popover container and is controllable with a flag.

Reply:
Any particular reason for this? Now `hover-area` and its associated styles are added to the generic popover component, although there is no other use case AFAICS.